Getting Started with Form (#1485)
DevExpress/devextreme-documentation: concepts/05 UI Components/Form/01 Getting Started with Form/00 Getting Started with Form.md

> The Form component displays fields from data objects as a collection of label-editor pairs and allows users to edit them. You can process the form data, validate it, and send it to a server.

Reviewer: My first red flag was the word "them" at the end of the first sentence. Is it "objects", or "fields", or "pairs"? We need to be careful with pronouns. It's safest when you refer to the closest noun, I guess. 

I'm also not sure what you mean by "process form data" in the second sentence - so my suggestion might not be 100% true to what you were trying to say: 

```suggestion
The Form component creates data entry UI for the underlying data object. The interface consists of automatically-arranged label-editor pairs that correspond to data fields. You can change data values, validate user input, and send resulting data to the server.
```

> The Form component creates data entry UI for the underlying data object. The interface consists of automatically-arranged label-editor pairs that correspond to data fields. You can change data values, validate user input, and send resulting data to the server.

Reviewer 2: ```suggestion
The Form component creates a data entry UI for the underlying data object. The interface consists of automatically-arranged label-editor pairs that correspond to data fields. You can change data values, validate user input, and send the resulting data to the server.
```

> This tutorial shows how to configure basic Form features. As a result, you will get a UI component that looks as follows:

Reviewer: I guess you guys use this template in all topics. I left a similar comment in "Getting Started with the Scheduler". 

The "red flag" for me here is the verb "get". It's on the list of "weak and boring" verbs in the "Misplaced Modifier" book. 

I'm also not sure about the phrase "basic features" - kind of makes our features sound not too exciting. I'd rather change it to "basics of ... configuration". 

```suggestion
This tutorial shows you the basics of **Form** component configuration. The preview below shows the expected result:
```

> This tutorial shows you the basics of **Form** component configuration. The preview below shows the expected result:

Reviewer 2: "shows you the basics" means what? What does it show? A tutorial usually explains something, gives information, or describes a specific functionality - images show or illustrate things. You also use **shows** in both sentences.

Author: Got it from Vladimir Abadzhev, he modified our template for 'getting started' tutorials. My initial option was something like

'This tutorial shows how to configure basic Form features. As a result, you will get a UI component that looks as follows:'

Please also take into account that the code in the line below displays a special frame with a working application:
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/33040934/106899898-f69ed080-6706-11eb-8a6f-8583992a8587.png)

Reviewer 2: This tutorial explains how to configure the **Form** component.
OR
This tutorial shows basic **Form** component configuration.

(in my opinion, I would leave out "The preview below shows the expected result" because the first sentence already states what the aim of this topic is)

<div class="simulator-desktop-container" data-view="/Content/Applications/20_2/GettingStartedWith/Form/index.html, /Content/Applications/20_2/GettingStartedWith/Form/index.js, /Content/Applications/20_2/GettingStartedWith/Form/index.css"></div>

> Refer to the following sections for details on each configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-form" target="_blank">getting-started-with-form</a>.

Reviewer 2: ```suggestion
Refer to the following sections for more information about each configuration step. The full code is available in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-form" target="_blank">getting-started-with-form</a>.
```
